045-InfraAsCode-Bicep/Student/Bicep-Challenge-13.md

```diff
@@ -0,0 +1,43 @@
+# Challenge 13 - Deploy an AKS Kubernetes Cluster
```

Comment:
Since challenge 12/13 are optional, can we mention them in the hack and readme?

Reply:
Hmm, I thought I had removed references to 12/13 being optional.  Can you provide a ref?